openbb_terminal/stocks/stocks_helper.py

```diff
@@ -227,14 +230,16 @@ def search(
         headers=["Name", "Country", "Sector", "Industry", "Exchange"],
         title=title,
     )
+    if export_to_file:
+        export_data(
+            export,
+            os.path.dirname(os.path.abspath(__file__)),
+            "search",
+            df,
+            sheet_name,
+        )
```

Comment:
I am not following the logic of this `export_to_file`.  If export=="", which it is by default, nothing gets saved.

The way we have our sdk is that exporting should not be handled through this.  Returning a df allows you to save, ie `openbb.stocks.search("Apple").to_csv("path/to/file"`

Reply:
Ah, you are right.  The underlying function does have that check, I should have noticed.  

I am not sure if I fully understand your second point as I see that as feature/benefit.  Is it going against the design methodology?  Is there an alternative method for retrieving the results into memory without hitting the file system?

Reply:
So it is against our design to have the export in the sdk model functions.  In this example, if you just call

df = openbb.stocks.search("Apple")

That will be in your memory locally.  If you want to save to a file you can using pandas built-ins.  

I might not be understanding what you wish to accomplish.  When you say you want it into memory, what are you hoping to do?

Reply:
I just pushed some commits to remove that redundant flag that I added.  Updated documentation as well.

The desire behind this pull request is to enable this particular function to actually return the results in dataframe format - if any are available -  exactly as you said: `df = openbb.stocks.search("Apple")`

Right now - from my understanding - It doesn't return anything.

Reply:
Yes that is correct.  You fixed that by returning the df.  We just dont need to have the export/sheet name in the model functions.

Reply:
Ah now I understand! (Hopefully).  

I pushed a commit that removes the export/sheet name in that function.  I adjusted and ran the tests to success.

Also redid my initial manual testing.

I also saw the mypy failures, and changed the return statements to give back an empty dataframe.   Similar to other functions in the same file.